tests/e2e/utils/data_fetcher/logs.py

```diff
@@ -133,6 +134,8 @@ def get_logs(
         Name of Lambda function to fetch logs for
     start_time : datetime
         Start date range to filter traces
+    expected_number_of_logs : Optional[int]
```

Comment:
```suggestion
    minimum_log_entries : Optional[int]
```

align with `TraceFetcher(minimum_traces=1)`, then we can have as 1. If you're using Log Entries you're expecting at least one entry, otherwise set it explicitly.